docs/src/user-manual/glossary.md

```diff
@@ -0,0 +1,368 @@
+
+There are many types of metadata, such as:
+
+* **Descriptive metadata**: Information that supports the identification and
+  discovery of a resource, describing the contents and their relationships to
+  other resources and entities. For example, the title of a document.
+* **Administrative metadata**: Information that supports the internal management
+  and governance of a resource, capturing its provenance, any related rights
+  statements or limitations, selection criteria, and/or local policies or
+  agreements impacting its use, acquisition, or disposition. For example, the
+  date a record was acquired by an archive.
+* **Preservation metadata**: Information necessary to manage and preserve a
+  resource over time including the documentation of any preservation actions
+  taken to ensure the authenticity of the resource, and the ongoing efforts to
+  ensure its preservation and usability over time, across space, and through
+  technological change. For example, documenting the process of producing a
+  checksum of a file when receiving it from an agency.
+* **Technical metadata**: Information that describes the technical properties
+  and characteristics of a resource required to render or process it. For
+  example, the bit depth of an audio **file**.
+* **Structural metadata**: Information that documents the relationship within
+  and among a resource, enabling end users to understand its structure and how
+  to navigate complex **objects**. For example, the ordering of individual TIFF
+  scans making up a book object.
+
+## Normalization
+
+A preservation processing **task** that creates a copy of a **file** in a
+different **format** based on a **preservation policy**, as part of a strategy
+to support long-term preservation and/or access.
+
+When normalization occurs for preservation (i.e. during **AIP** creation), the
+goal is generally to avoid technical obsolescence, and target formats are
+typically chosen based on open licenses and specifications, lossless
+compression, ubiquity of support and use, and community consensus on best
+practice. When normalization is performed for access (i.e. during **DIP**
+creation), target formats are typically chosen based on file size and
+compression levels, ease of transmission, and ubiquity of support and adoption
+(particularly on the public web).
+
+## Object
+
+An **intellectual entity** that describes something of enduring value, and under
+consideration for preservation. There is at least one object in every
+**package**, and each object can contain zero or more **files**. An object may
+be described by a **producer** using any desired descriptive **metadata** that
+is meaningful to the designated community.
+
+## Package
+
+An **intellectual entity** describing a type of collection, composed of a set of
+**files** and related **metadata**, assembled together for a particular purpose.
+A package will typically contain one or more **objects**, zero or more
```

Comment:
By the definition of "Object" above a package must *always* contain one or more objects.

Reply:
sure... but adding "must" and "may" gets a bit complex. I just removed the word "typically" - does that work?

Reply:
Sure, sounds good. :+1: